autoscheduler/frontend/assets/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "",
```

Comment:
I think we should go ahead and add a name, since it isn't too hard, and it'll help us achieve full PWA if we ever want to